Mathlib/Combinatorics/SimpleGraph/DegreeSum.lean

```diff
@@ -5,6 +5,7 @@ Authors: Kyle Miller
 -/
 import Mathlib.Combinatorics.SimpleGraph.Basic
 import Mathlib.Algebra.BigOperators.Basic
+import Mathlib.Data.Finset.Sym
```

Comment:
Why do we need this extra import?

Reply:
It brings in `Sym2.instFintype`

Reply:
Hmm okay sure, but why do we suddenly need `Sym2.instFintype` here?

Reply:
Ah, is it because you can't use the `Quotient` instance anymore? Worth mentioning in the description.

Reply:
Isn't that typeclass version `HasEquiv`?

Reply:
No, [that's](https://github.com/leanprover/lean4/blob/b0fe9d6cdca82723134d61f8fe23a8c674f308b7/src/Init/Core.lean#L338-L341) a notation typeclass.

I mean this:
```lean
class IsEquivalence {α : Type*} (r : α → α → Prop) : Prop where
  equivalence : Equivalence r
```